docs/source/Linux_Install_Guide.rst

```diff
@@ -202,14 +200,14 @@ rocWMMA has several test suites that can be built:
 - Unit tests
 
 DLRM tests cover a Deep Learning Recommendation Model implemented with rocWMMA.
```

Comment:
I would change this to:
DLRM tests cover the dot product interactions between embeddings used in DLRM